src/txmempool.h

```diff
@@ -63,9 +63,10 @@ class CTxMemPoolEntry
     size_t nModSize; //! ... and modified size for priority
     size_t nUsageSize; //! ... and total memory usage
     int64_t nTime; //! Local time when entering the mempool
-    double dPriority; //! Priority when entering the mempool
```

Comment:
> It was necessary to distinguish between the two stored priorities and heights. The original "entry" priority and height and the latest calculated "cached" priority and the height that was calculated at.

I guess if the rename had been introduced when it was necessary instead of here I wouldn't have complained.

> It seemed logical to me to leave those changes in given that we didn't know whether the second half would be merged or not and given that they are more accurate names anyway.

I'm confused, how are CTxMemPoolEntry::nHeight and CTxMemPoolEntry::dPriority "less accurate" than CTxMemPoolEntry::entryHeight and CTxMemPoolEntry::entryPriority ?